### PR TITLE
Updates after changing the transport seam

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -196,4 +196,17 @@ NServiceBus version 6 is no longer scanning nested directories for assemblies. Y
...
+
+If your are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
+
+## Transport seam
+
+`IDispatchMessages` have been obsoleted and is replaced by `IPushMessages`. The interfaces are basically equivalent so if you're implementing your own transport you should be able to just implement the new interface.
+
+### Corrupted messages
+
+The core will now pass the error queue address to the transport to make it easier to handle corrupted messages. If corrupted message is detected the transport is expected to move the message to the specified error queue.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: > The core will now pass the error queue address to the transport

how?

Comment (andreasohlund) [inline nservicebus/upgrades/5to6.md]: Reworded it, does it make more sense now?

**Comment by SimonCropp** [comment]
@andreasohlund can this be rebased. also is this a WIP that is waiting for V6 to be released?

**Comment by andreasohlund** [comment]
I'll rebase, this one doesn't need to wait for v6. Should I merge it after the rebase?

File: nservicebus/upgrades/5to6.md [hunk 2]
```diff
@@ -196,4 +196,17 @@ NServiceBus version 6 is no longer scanning nested directories for assemblies. Y
...
+In version 6 the API has been changes to a "Exclude a list" approach. See [Assemblies to scan](/nservicebus/hosting/assembly-scanning.md#assemblies-to-scan) for more information.
+
+
+## RepeatedFailuresOverTimeCircuitBreaker has been made internal
+
+If your are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
+
+## Transport seam
+
+`IDispatchMessages` have been obsoleted and is replaced by `IPushMessages`. The interfaces are basically equivalent so if you're implementing your own transport you should be able to just implement the new interface.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove `basically`

File: nservicebus/upgrades/5to6.md [hunk 3]
```diff
@@ -196,4 +196,17 @@ NServiceBus version 6 is no longer scanning nested directories for assemblies. Y
...
+
+### Corrupted messages
+
+The core will now pass the error queue address to the transport via the `IDispatchMessages.Init(PushSettings)` API. The transport is expected to move the corrupted message to the specified error queue when detected.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: > `IDispatchMessages.Init(PushSettings)` API

will there be more doco for this?

Comment (andreasohlund) [inline nservicebus/upgrades/5to6.md]: I'd say yes, as soon as the seam is stable we should document it? (@distantcam is updating RabbitMQ as we speak)

**Pull request merged**